packages/office-ui-fabric-react/src/components/Button/CommandButton/CommandButton.tsx

```diff
@@ -27,9 +28,10 @@ export class CommandButton extends BaseComponent<IButtonProps, {}> {
   public render() {
     return (
       <BaseButton
-        classNames={ CLASS_NAMES }
+        { ...this.props }
+        classNames={ assign({}, CLASS_NAMES, this.props.classNames) }
```

Comment:
That should still work with object spread though. They are equivalent and is more compact:  https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-1.html